WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -185,22 +203,23 @@ class AccountSettingsService {
         settings = accountSettingsWithID(self.userID)
     }
 
-    @discardableResult fileprivate func applyChange(_ change: AccountSettingsChange) throws -> AccountSettingsChange {
-        var reverse: Result<AccountSettingsChange, Error>! = nil
-        coreDataStack.performAndSave { context in
+    @discardableResult fileprivate func applyChange(_ change: AccountSettingsChange) async throws -> AccountSettingsChange {
```

Comment:
Does this need to be `fileprivate`?

```suggestion
    @discardableResult private func applyChange(_ change: AccountSettingsChange) async throws -> AccountSettingsChange {
```